CMSgov/easi-app
# [EASI-2021] Remove email notification feature flag

# EASI-2021

## Changes and Description

- Removes code related to handling feature flag
- Removes `shouldSendEmail` from graphql schema for all action inputs
  - Notification email only sends if `notificationRecipients` object is included in request
  - `shouldSendEmail` is still included in frontend as a Formik field for field validation
- Updates component frontend tests to use new mock CEDAR contacts
  - `<CedarContactSelect />` - src/components/CedarContactSelect/index.test.tsx
  - `<AdditionalContacts />` - src/components/AdditionalContacts/index.test.tsx
- Updates Cypress tests that were previously skipped due to feature flag
  - cypress/e2e/systemIntake.cy.js
  - cypress/e2e/governanceReviewTeam.cy.js



## How to test this change

- Confirm that code related to `notifyMultipleRecipients` feature flag has been removed
- Test GRT actions with and without email to confirm that everything still works as expected after removing `shouldSendEmail` from schema
  - Emails can be tested using the [MailCatcher UI](http://127.0.0.1:1080/)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
